src/components/editor/FormWrapper.jsx

```diff
@@ -29,7 +31,9 @@ class FormWrapper extends Component{
                     return (<p key={index}><b>{pt.propertyLabel}</b>: <i>resource</i> type</p>)
                   }
                   else if (pt.type == 'lookup'){
-                    return (<p key={index}><b>{pt.propertyLabel}</b>: <i>lookup</i> type</p>)
+                    return(
```

Comment:
I made issue #225 for the refactoring I suggested to you and @SaravShah the other day for     isItQA   and isItModal  work.  That *still* won't address when there is a combination of a valueTemplateRef and a useValuesFrom, etc.  Ugh.   Maybe the research I intend to do today to see what triples are generated from different cases will be helpful.